packages/browser/src/core/http-cookies/__tests__/index.test.ts

```diff
@@ -7,6 +7,63 @@ async function sleep(delayMS: number): Promise<void> {
 }
 
 describe('HTTPCookieService', () => {
+  it('handles different combinations of origin and url path', async () => {
+    // HTTPCookieService constructor uses `new URL(path, origin).href to normalize URLs
+
+    // if the user passes a slash prefix
+    expect(
+      HTTPCookieService.urlHelper({
+        origin: 'http://localhost:8080/',
```

Comment:
Origin is an optional argument. The intention is for the user normally just pass the required reqs "renewUrl" and "clearUrl" and the SDK then figures out the correct domain on it's own. `origin` is passed here more for tests like this.